### DISCUSSION TITLE
getActions() method how to work

### DISCUSSION THREAD

**Discussion by R1NEJ0** [filamentphp/filament#2276] [Help] [answered]
Hi everyone,

I'm trying to add a modal with getActions(), but I can't figure out how can I run it.

I need display a modal with two buttons.... Thanks in advance.

I though this method it's the correct because in documentation I can read..:

"Actions" are buttons that are displayed next to the page's heading, and allow the user to run a Livewire method on the page or visit a URL.

**Comment by danharrin** [comment, marked as answer]
What code have you tried?

**Reply by R1NEJ0** [reply]
I'm quite noob using Livewire, I don't how to use it.... so never mind thanks por quick response.

I'm going to try usin wire-elements/modal.

Thanks you!

**Reply by danharrin** [reply]
I'd like to help you use the built in modals if you want to post the code :)

**Reply by R1NEJ0** [reply]
At the moment there is no code, I was figuring out how to build a custom modal to show two buttons.

😄

**Reply by R1NEJ0** [reply]
Finally I did it with wire-elements/modal, it's been tricky because the CSS of wire-elements/modal doesn't fit properly with filament's CSS
